polaris-react/scripts/build-consumer.js

```diff
@@ -25,7 +25,7 @@ const projectPolarisDir = resolve(
 const files = [
   'package.json',
   'README.md',
-  'LICENSE.md',
+  '../LICENSE.md',
```

Comment:
As LICENCE on longer exists in the package directory I bet it won't be published to npm anymore - check what gets created in the tarball when you do `npm pack` in the polaris-react directory, and compare it to https://unpkg.com/browse/@shopify/polaris@9.2.2/

Reply:
As it is a custom licence (rather than standard MIT / apache2 / whatever) and the licence field says `"license": "SEE LICENSE IN LICENSE.md",`, that really needs to be part of the distributed package.

Reply:
Yeah, I agree and I was going to handle it in a separate issue/PR but I'll just add the commits here.
The CHANGELOG.md wasn't being added either so I included that